### DISCUSSION TITLE
why ubuntu working in utm use only 50gb eventhough i allotted 100gb space in my macbook

### DISCUSSION THREAD

**Discussion by MohamedNadeemIT** [utmapp/UTM#3605] [UTM Help (macOS)]

**Comment by MohamedNadeemIT** [comment]
Hard drive specifically

**Comment by adespoton** [comment]
100GB is the maximum space the volume can take.  UTM uses QCOW2 volumes, which only take up as much space as the data they're currently storing, expanding in size until they hit the hard limit (in this case, 100GB).

**Reply by MohamedNadeemIT** [reply]
The problem is I allocated 100GB but I can only see 50 GB available in the drive. I don’t understand where the remaining 50GB is
